bro-spinn/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/brospinn/SpinnService.kt

```diff
@@ -63,7 +63,7 @@ class SpinnService(
 
         val transaksjonId = Key.UUID.les(UuidSerializer, json)
 
-        val forespoerselId = RedisKey.of(transaksjonId.toString(), DataFelt.FORESPOERSEL_ID)
+        val forespoerselId = RedisKey.of(transaksjonId, DataFelt.FORESPOERSEL_ID)
```

Comment:
For disse kan vi vel få in-flight problemer, altså der de settes som string i gammel versjon, deretter deployes ny versjon som ikke klarer å lese ut. Usikker på om det er verdt å legge til en fallback til gammel variant, og deretter deploye endelig versjon uten fallback, det kommer litt an på hvor store evt konsekvenser blir hvis lesing feiler...

Reply:
Det blir ikke problemer fordi den underliggende nøkkelen blir ikke endret. Det er bare interfacet som vi bruker i koden som endrer seg.